src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,77 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+
+  margin-top: 8px;
+  margin-bottom: 8px;
+  gap: 17px;
+
+  &-amount {
+    display: flex;
+
+    justify-content: flex-end;
+    align-items: flex-end;
+    height: 16px;
+    width: 53px;
+    margin: 6px 0 10px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 16px;
+    text-align: right;
+
+    color: $main-text-color;
+  }
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  margin: 8px 0 12px;
+  gap: 4px;
+
+  &__star {
+    display: flex;
+    background-image: url(/src/images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:nth-child(-n + 4) {
+      background-image: url(/src/images/star-active.svg);
+    }
```

Comment:
This part of the code is not very maintainable and scalable. If you want to change the number of active stars, you have to change the CSS code. It would be better to control this with a modifier class like `stars__star--active` and add this class to the active stars in your HTML.